nixos/common/flake.nix

```diff
@@ -42,6 +57,9 @@ in
     };
   };
   config = lib.mkIf (cfg.flake != null) {
+    nix.settings.nix-path = cfg.flake.inputs.nixpkgs;
```

Comment:
This syntax works as intended? Just reviewing the code atm, haven't tested but would have expected something like
`nix.settings.nix-path = "nixpkgs=${cfg.flake.inputs.nixpkgs}";`

Reply:
Good call.